backend/src/test/java/com/github/boltzmann/biokiste/backend/controller/GetAllPublicBoxNamesAndIdsTest.java

```diff
@@ -0,0 +1,74 @@
+package com.github.boltzmann.biokiste.backend.controller;
+
+import com.github.boltzmann.biokiste.backend.dto.OrganicBoxDto;
+import com.github.boltzmann.biokiste.backend.model.OrganicBox;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+class GetAllPublicBoxNamesAndIdsTest extends CrudTestWithLogIn {
+
+
```

Comment:
Es ist jetzt hier ein bisschen schwer nachzuvollziehen, aber mischt du nicht unit mit integrationstest in dieser test class?

Reply:
Nach längerem Nachdenken .. hast du recht. Aber die Tests finde ich toll, es hilft mir.